.github/workflows/go.yml

```diff
@@ -0,0 +1,29 @@
+name: Go
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+    
+    - name: Install GolangCI-Lint
+      run: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s $version
+      env:
+        version: v1.19.1
+
+    - name: Lint
+      run: ./bin/golangci-lint run
+          
+    - name: Test
+      run: |
+        go test ./...
```

Comment:
Might want to add the `-v` flag for more verbose output.

Reply:
Thanks. I added.